src/Api/Webhook.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Unione\Api;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Unione\UnioneClient;
+use Webmozart\Assert\Assert;
+
+/**
+ *  This class for sending  Webhook.
+ */
+class Webhook
+{
+    /**
+     * The UnioneClient client.
+     *
+     * @var UnioneClient
+     */
+    private UnioneClient $client;
+
+    /**
+     * @param UnioneClient $client
+     */
+    public function __construct(UnioneClient $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Sets or edits a webhook.
+     *
+     * @param array {
+     *              'url': string
+     *              'events': array {'email_status': array}
+     *            } $params the request parameters
+     *
+     * @return array           API response
+     * @throws GuzzleException
+     * @see  https://docs.unione.io/en/web-api-ref#webhook-set
+     */
+    public function set(array $params): array
+    {
+        Assert::string($params['url'], 'The url must be a string. Got: %s');
+        Assert::isNonEmptyList($params['events']['email_status'], 'The email_status must be a not empty array. Got: %s');
```

Comment:
I would clarify that `email_status` should contain at least one of the events to subscribe.